docs/workflows/pr.md

```diff
@@ -55,11 +55,11 @@
 1. PR 在合并当前 master 到 PR 之前无法自动合并
 2. PR 合并当前 master 到 PR 导致 `status/can-merge` 标签消失
 
-**第一个问题使用 [ti-community-tars](./../plugins/tars.md) 自动更新就可以解决。第二个问题因为我们可以识别到使用 GitHub 更新按钮更新 master 到 PR 的提交的 committer 为 `web-flow`，所以可以根据 committer 来判断是否信任该提交。**
+**第一个问题使用 [ti-community-tars](plugins/tars.md) 自动更新就可以解决。第二个问题因为我们可以识别到使用 GitHub 更新按钮更新 master 到 PR 的提交的 committer 为 `web-flow`，所以可以根据 committer 来判断是否信任该提交。**
 
 ### 大型仓库推荐使用 rerere 
 
-我们使用 [ti-community-tars](./../plugins/tars.md) 并且打开  Require branches to be up to date before merging 可以解决 PR 合并问题，但是对于大型仓库来说，这样会极大的浪费测试资源。所以推荐使用 rerere 解决该问题，详见：[rerere](../components/rerere.md)。
+我们使用 [ti-community-tars](plugins/tars.md) 并且打开  Require branches to be up to date before merging 可以解决 PR 合并问题，但是对于大型仓库来说，这样会极大的浪费测试资源。所以推荐使用 rerere 解决该问题，详见：[rerere](../components/rerere.md)。
```

Comment:
This path `../components/rerere.md` looks different.

Reply:
If the path is right, it's OK.

Reply:
Good Catch!

Fixed.